.github/workflows/blobby.yml

```diff
@@ -4,19 +4,19 @@ on:
   push:
     branches: [main]
     paths:
-      - "actor/blobby/**"
+      - "actor/blobby/actor/**"
+      - "actor/blobby/testing/**"
     tags:
       - "blobby-v*"
   pull_request:
     branches: [main]
     paths:
-      - "actor/blobby/**"
+      - "actor/blobby/actor/**"
```

Comment:
Is there something at `actor/blobby`  root that you don't want this to trigger on?

Reply:
README. Plus just being explicit :)

Reply:
Read the README just my paranoia that we'll change source at the actor root (e.g. add a Makefile) and forget about this. Minor nit, just wanted to make sure I wasn't missing a trick.